### PR TITLE
prepare for v0.11.0 release

### PR DESCRIPTION
**Pull request by rfratto**
Release testing looks good. Found one hidden problem from #336 which is fixed by disabling the unneeded CGO_ENABLED=1 in agentctl builds.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -143,9 +143,9 @@ endif
 
 cmd/agentctl/agentctl: cmd/agentctl/main.go
 ifeq ($(CROSS_BUILD),false)
-	CGO_ENABLED=1 go build $(GO_FLAGS) -o $@ ./$(@D)
+	CGO_ENABLED=0 go build $(GO_FLAGS) -o $@ ./$(@D)
 else
-	@CGO_ENABLED=1 GOOS=$(GOOS) GOARCH=$(GOARCH) GOARM=$(GOARM); $(seego) build $(CGO_FLAGS) -o $@ ./$(@D)
+	CGO_ENABLED=0 GOOS=$(GOOS) GOARCH=$(GOARCH) GOARM=$(GOARM) go build $(GO_FLAGS) -o $@ ./$(@D)
```
Comment (rfratto) [inline Makefile:148]: @56quarters heads up, found this in release testing. Adding config validation imported promtail from agentctl, which accidentally added a dependency on libsystemd-dev being installed, and was causing `make agentctl-image` to fail locally. The CI builds a cross-platform image so it wasn't caught there. 

agentctl doesn't need CGO, so I've just disabled it here to fix the problem.

**Comment by rfratto** [comment]
There is a `0.10.0` in the Makefile for the build image used for packaging RPMs/DEBs, but it's unrelated to Agent v0.10.0

File: Makefile (hunk 1 above)
Comment (56quarters) [reply Makefile:148]: Gah, sorry about that.

Comment (rfratto) [reply Makefile:148]: No worries!

**Review by 56quarters — APPROVED** [review]

**Review by mattdurham — APPROVED** [review]

**Pull request merged**